modules/local/combine_tsv.nf

```diff
@@ -0,0 +1,31 @@
+process COMBINE_TSV {
+    tag "$meta.id"
+    label 'process_high'
+
+    // Using bioawk as we already use that for CONVERT_DEPTHS and does same thing
```

Comment:
You've borrowed from mag haven't you ;) that comment looks familiar

There is a official bioawk module now though - why not use it...